File: adopt-me-front/src/api/article/response/ArticleDto.ts
import {ArticleTypeEnum} from "../ArticleTypeEnum";

export interface ArticleDto{
    id:string
    title:string
    content:string
    attachmentId?:string
    type:ArticleTypeEnum
    createdOn: Date,
    createdById: string
    updatedOn: Date
    deletedOn?: Date,
    deletedById?: string
}

File: adopt-me-backend/modules/article-module/article-rest/src/main/java/pl/adoptme/adopt/me/article/ArticleDto.java
package pl.adoptme.adopt.me.article;


import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;


@Getter
@Setter
@Builder
public class ArticleDto {
    private String id;
    private String title;
    private String content;
    private String attachmentId;
    private ArticleType type;
    private LocalDateTime createdOn;
    private String createdById;
    private LocalDateTime updatedOn;
    private LocalDateTime deletedOn;
    private String deletedById;
}
